src/Symfony/Component/Emoji/EmojiTransliterator.php

```diff
@@ -85,12 +85,12 @@ public function createInverse(): self
 
     public function getErrorCode(): int|false
     {
-        return $this->transliterator?->getErrorCode() ?? 0;
+        return $this->transliterator->getErrorCode();
```

Comment:
what happens if you call this with an uninitialized property (by calling that before using `transliterate`) ?

Reply:
You're right, I was missing the unset-safe role of the operator. That's reverted

Reply:
I don't think `?->` is unset-safe, is it?

Reply:
I wasn't aware of this behavior, but yes it is: https://3v4l.org/MJAaP. Also nullsafe can be omitted if null coalesce is used: https://3v4l.org/Gvb4PQ

Reply:
No it's not. `??` is, which is what your snippet relies on: https://3v4l.org/lIQHm

Reply:
Ah yes, I got confused by the fact that the Transliterator code uses null coalesce, sorry.

Reply:
see #58623